src/app/admin/subjects/subjects-create-modal/subjects-create-modal.component.html

```diff
@@ -1,6 +1,7 @@
-<form class="input-wrapper" [formGroup]="addSubject">
+<form class="input-wrapper" [formGroup]="addSubject" autocomplete="off">
     <mat-form-field class="example-full-width">
```

Comment:
I suggest using appropriate css classes (not 'example-full-width' but 'full-width')